api/script/index.js

```diff
@@ -16,8 +16,10 @@ var wrapper = {
     },
     'organization': function(msg, $meta) {
         return this.bus.importMethod('customer.organization.graphFetch')(msg, $meta).then(result => {
-            let organization = result.organization;
-            return {items: organization.map(v => ({ type: 'organization', value: v.id, display: v.title }))};
+            let organizations = result.organization.filter((obj, position, arr) => {
```

Comment:
If we want to remove duplicates, it is best this to be done in the customer module, so that each place that needs the duplicates removed can use the same code. Also best place to remove duplicates is most likely to be the stored procedure.

Also is it at all valid for duplicates to be returned?